HW4/process_001.py

```diff
@@ -0,0 +1,44 @@
+"""" Асинхронная загрузка с использованием модуля asyncio """
+import argparse
+import asyncio
+import aiofiles
+import aiohttp
+import time
+
+# urls = [
+#         'https://ssl.robocup.org/wp-content/uploads/2021/12/20160629-IMG_0283_1.jpg',
+#         'https://s.studiobinder.com/wp-content/uploads/2021/01/Best-Black-and-white-pictures.jpg',
+#         'https://burst.shopifycdn.com/photos/full-blue-moon.jpg'
+#         ]
+
+async def download(url):
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url, ssl=False) as response:
+            filename = url.strip().split('/')[-1].strip()
+            f = await aiofiles.open(filename, mode='wb')
+            await f.write(await response.read())
+            await f.close()
```

Comment:
А почему бы тут просто не использовать синхронный менеджер контекстов?
```suggestion
            with open(filename, "wb") as f:
                f.write(response.content)
```